sites/main-site/src/content/blog/2024/seqera-containers-part-1.mdx

```diff
@@ -0,0 +1,453 @@
+    -   [Better transparency](#trust-and-transparency)
+
+With the new Seqera Containers setup we should be able to address these problems, as well as provide the additional features.
+
+:::tip{title="What's changing"}
+A detailed description of the proposed mechanism will come in part 2 of this blog post.
+Here are the key points:
+
+-   Developers will _only_ need to edit the conda `environment.yml` (no process `container`)
+-   Container images will be built by Wave and stored in Seqera Containers
+-   Build logs and source files will be shown on the nf-core website module pages
+-   Conda lock-files will be used for Conda users and CI tests
+-   Software releases will be automatically bumped in modules, using [Renovate](https://renovatebot.com/)
+-   There should be almost no change in how end-users run nf-core pipelines
+    :::
+
+:::info{.fa-calendar-lines-pen title="Timeline" collapse}
+
+This move has been in the works for over a year now.
+It started with discussions between nf-core maintainers and Seqera developers about what the community needed.
+Seqera's open source [Wave](https://seqera.io/wave/) container tool brings convenience, but long term storage of container images and stable container URIs were identified as hard requirements.
+In response, Seqera developed _Seqera Containers_, a free community resource with hosting infrastructure funded by AWS.
+This service was launched in spring 2024.
+
+The adoption of Seqera Containers in nf-core was initially raised in the nf-core steering group, followed by discussion in the
+core team and then maintainers team (see the [nf-core governance structure](/governance)).
+At each step, the discussion informed the planned infrastructure and setup for this change, as well as the development of Seqera Containers and Wave.
+
+Several steps are still needed to complete this migration:
+
+-   nf-core/modules automation for fetching images and pinning conda-lock files and `meta.yml` references
+-   nf-core/tools tooling for pipeline config file generation, with pipeline template update
+-   Bulk update of nf-core/modules to use Seqera Containers
+-   Update of nf-core pipelines to use the new modules
+
+If all goes to plan, we hope to have all modules updated by the end of 2024.
+This work will be tracked in GitHub issues as usual,
+starting with [nf-core/modules#5832](https://github.com/nf-core/modules/issues/5832).
+This issue has already had a substantial amount of discussion, much of which has informed this blog post.
+
+It's not too late to add your feedback! So if you have any questions or ideas, let us know on Github or Slack.
+
+Note that images from Seqera Containers can already be used in nf-core/modules, with the old syntax of using a `container` declaration (see [example](https://github.com/nf-core/modules/blob/b6b54f3929b0ba4a7c02a49191308ce1d8351f0d/modules/nf-core/bedtools/genomecov/main.nf#L6-L8)).
+
+:::
+
+## Comparing BioContainers and Seqera Containers
```

Comment:
A bit weird there is no description of what./how biocontainers works, when this title is 'comparing...', just what biocontainers _doesn't_ do

Reply:
Yeah this is the victim of moving stuff around a bit. We do introduce BioContainers in the introduction already.

I'll try moving this down to here and leave just a very short bit in the intro.

Reply:
Ok had a stab at doing this in 706fa025